src/Spryker/Zed/CompanyBusinessUnit/Persistence/CompanyBusinessUnitRepository.php

```diff
@@ -20,6 +20,11 @@
  */
 class CompanyBusinessUnitRepository extends AbstractRepository implements CompanyBusinessUnitRepositoryInterface
 {
+    /**
+     * @var string
+     */
+    protected const TABLE_JOIN_PARENT_BUSINESS_UNIT = 'parentCompanyBusinessUnit';
```

Comment:
```suggestion
    protected const TABLE_JOIN_PARENT_COMPANY_BUSINESS_UNIT = 'parentCompanyBusinessUnit';
```
I'm not sure if we have a convention for how to name this constant. But wouldn't this make it more clear?

Reply:
Fixed